restful.module

```diff
@@ -339,7 +339,14 @@ function restful_delivery($var = NULL, $method = 'format') {
       ->getFormatterManager();
     $formatter_manager->setResource($resource);
     $plugin_definition = $resource->getPluginDefinition();
-    $formatter_name = isset($plugin_definition['formatter']) ? $plugin_definition['formatter'] : NULL;
+    if ($request->getMethod() == RequestInterface::METHOD_OPTIONS) {
```

Comment:
I'm not sure about this restriction - wouldn't you want to get the OPTIONS info in `json-api` format?